charms/layers/java/reactive/java.py

```diff
@@ -33,23 +34,11 @@ def install():
         oracle.installoracle()
         set_state('java.installed')
     else:
-        hookenv.satus_set('blocked','Only Oracle JDK and OpenJDK are supported!')
+        status_set('blocked', 'Only Oracle JDK and OpenJDK are supported!')
 
 
 # Special handler for openjdk because openjdk 8 needs another repo
 # Also sets the JAVA_HOME path
 @when_any('apt.installed.openjdk-6-jre-headless', 'apt.installed.openjdk-7-jre-headless', 'apt.installed.openjdk-8-jre-headless')
 def openjdk_install():
-    with open(os.path.join(os.path.expanduser('~'), '.bashrc'), 'a+') as file:
-        if 'JAVA_HOME' not in file:
```

Comment:
Are you sure you can remove this? A lot of applications depend on this to be present..